aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/SRPHelper.kt

```diff
@@ -34,7 +34,7 @@ import org.jetbrains.annotations.TestOnly
  * https://github.com/aws-amplify/aws-sdk-android/blob/main/aws-android-sdk-cognitoidentityprovider/src/main/java/com/amazonaws/mobileconnectors/cognitoidentityprovider/CognitoUser.java#L3587
  * SRP requires Kotlin version 1.5+, and minSDK version 24.
  */
-class SRPHelper(private val password: String) {
+class SRPHelper(private val password: String = "") {
```

Comment:
Let's make SRP custom auth action pass in empty string explicitly instead of adding a default value here. Default empty value can cause a miss in SRP action.